scripts/python/test_get_slack_support_messages.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from unittest.mock import MagicMock, patch
+
+import get_slack_support_messages
+# TODO: This mock needs fixing before tests can be implemented
+
+
+@patch("slack_sdk.WebClient.__new__", new=MagicMock(conversations_history=MagicMock(has_more=False)))
+@patch("os.environ", new=MagicMock(SLACK_BOT_TOKEN="token"))
+class TestGetSlackSupportMessages(unittest.TestCase):
+
+    # TODO: requires deeply nested mocking to smoke test due to infinite loop.
+    def ignore_test_main_smoke_test(self):
```

Comment:
thought it was better to have the test file here with a few comments explaining why we're not testing at the moment.

TLDR; 
I can't figure out how to mock nested structures and don't think the time spent working it out will be worth it. Better left to be decided when/if we refactor parts of the codebase